src/app/modules/gb-side-panel-module/accordion-components/gb-tree-nodes/gb-tree-nodes.component.ts

```diff
@@ -184,129 +170,6 @@ export class GbTreeNodesComponent implements OnInit, AfterViewInit, AfterViewChe
     }
   }
 
-  /**
```

Comment:
just wondering are the functions such as removeFalsePrimeNgClasses, clearFilter completely removed or they are reused somehow in the tree-search component?